.vscode/settings.json

```diff
@@ -96,5 +96,6 @@
   "cSpell.words": ["Discoverability"],
   "typescript.tsdk": "common/temp/node_modules/typescript/lib",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+  "eslint.experimental.useFlatConfig": true
```

Comment:
want to make sure eslint warnings shows up correctly in vscode

Reply:
Honestly can't tell since eslint plugin even on a normal basis is broken for my vscode. Can verify linting is working from the cli. I added this because of the README instruction here: https://github.com/iTwin/eslint-plugin. Lmk if you want me to remove, will do so. Thanks.

Reply:
you shoudl see squiggly lines when theres a linter error
make sure you have the vscode eslint extension installed
change some code so you have a linter error

<img width="1283" alt="image" src="https://github.com/iTwin/appui/assets/11051042/b3d20f2b-e92c-4fb5-a178-d832330768d6">

if you go to the output tab, change the task to eslint and you should hopefully see some errors in there

Reply:
Gotcha, it looks like it cannot find the config file it is looking for. I imagine if this is because of the monorepo rush setup we have: 
![image](https://github.com/iTwin/appui/assets/98850418/3a4ca7d1-ae52-44fd-b867-d2deb7abe58b)

Reply:
Plz get that working then. Might need to mess with vscode eslint settings

Reply:
working now